examples/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumb/types.ts

```diff
@@ -0,0 +1,62 @@
+import { AnyChildren } from '@ballerine/ui';
+
+export type BreadcrumbState = 'idle' | 'warning' | 'completed';
+export type BreadcrumbElements = 'wrapper' | 'outer' | 'inner';
+
+export interface BreadcrumbsOuterProps {
+  className?: string;
+  children?: AnyChildren;
+}
+
+export interface BreadcrumbsInnerProps {
+  className?: string;
+  icon?: React.ReactNode;
+}
+
+export interface BreadcrumbsWrapperProps {
+  className?: string;
+  children?: AnyChildren;
+}
+
+export interface BreadcrumbContext {
+  isActive: boolean;
+  elementsProps: {
+    outerProps: BreadcrumbsOuterProps;
+    innerProps: BreadcrumbsInnerProps;
+    wrapperProps: BreadcrumbsWrapperProps;
+  };
```

Comment:
If we move forward with React dot notation for either the breadcrumb item or each element (`outer`, `inner`, etc.) it would make a lot of sense for the elements to get their props directly, let me know if some code snippets would help. 🙂

Reply:
I would like to see snippets.